shell/config/table-headers.js

```diff
@@ -446,13 +446,26 @@ export const SUB_TYPE = {
   width:    120,
 };
 
+export const EVENT_TYPE = {
+  name:     'type',
+  labelKey: 'tableHeaders.type',
+  value:    'eventType',
+  sort:     'eventType',
+};
+
 export const STATUS = {
   name:     'status',
   labelKey: 'tableHeaders.status',
   value:    'status',
   sort:     ['status'],
   width:    175
 };
+export const LAST_SEEN_TIME = {
+  name:     'lastSeen',
+  labelKey: 'tableHeaders.lastSeen',
+  value:    'lastSeen',
+  sort:     ['lastTimestamp']
```

Comment:
One minor adjustment, imo, so take it with a grain of salt:

instead of `sort: ['lastTimestamp']` i would do `sort: 'lastTimestamp:desc'`. With this we would have the most recent first.

Note: apparently if you do `sort: ['lastTimestamp:desc']` it messes up the formatting of the value, so I just tried as a string and it worked.